python/setup.py

```diff
@@ -36,7 +36,7 @@ def package_files(directory):
           'jpype-stubs': jpype_stubs_paths,
           'org-stubs': org_stubs_paths
       },
-      install_requires=['JPype1==1.3.0', 'Deprecated==1.2.13'],
+      install_requires=['JPype1==1.4.0', 'Deprecated==1.2.13'],
```

Comment:
To make it still compatible with python 3.6:
```suggestion
      install_requires=['JPype1>=1.3.0,<=1.4.0', 'Deprecated==1.2.13'],
```